Microsoft.Toolkit.Uwp.UI.Controls/MasterDetailsView/MasterDetailsView.cs

```diff
@@ -152,6 +170,7 @@ private void OnLoaded(object sender, RoutedEventArgs e)
                     _frame.Navigating -= OnFrameNavigating;
                 }
 
+                _navigationView = this.GetAscendants().FirstOrDefault(p => p.GetType().FullName == "Microsoft.UI.Xaml.Controls.NavigationView");
```

Comment:
Should we also support the platform NavView?

Reply:
What's the recommended control moving forward? I figured it was going to be the WinUI version.

Reply:
Good question. On second thought, let's stick with this as the back button doesn't exist on older sdks